dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM ubuntu:24.04 as apt-requirements
+
+# Install APT requirements.
+RUN --mount=target=/requirements,type=bind,source=requirements <<EOF
+
+    set -ex
+
+    apt update
+
+    apt --yes --no-install-recommends install $(grep -vE "^\s*#" /requirements/requirements.system.txt  | tr "\n" " ")
+
+    apt clean && rm -rf /var/lib/apt/lists/*
+
+EOF
+
+# Compile Doxygen.
```

Comment:
OK. Just wondering if we could rely on `apt` to install `Doxygen` for us? Because we're on `ubuntu:24.04`, the installed version would be `1.9.8`, which, would be good (?).

Reply:
Ah, OK, I see, in the version you would like `1.10.0`. But perhaps `1.9.8` would be fine?

Reply:
I guess 1.9.8 could be fine, but since we're going to use cutting edge C++ features, I prefer to stick with the most recent Doxygen release 😉

Reply:
OK to keep it this way!